crates/nu-std/std/random/mod.nu

```diff
@@ -0,0 +1,19 @@
+# for examples
+alias "random dice" = dice
+
+# Generate a random dice roll
+@search-terms "generate" "die" "1-6"
+@example "Roll 1 dice with 6 sides each" { random dice }
+@example "Roll 10 dice with 12 sides each" {
+    random dice --dice 10 --sides 12
+}
+export def dice [
+    --dice = 1  # The amount of dice being rolled
+    --sides = 6  # The amount of sides a die has
```

Comment:
we need to ensure those values are positive, especially `--sides`

Reply:
Ah well spotted!